tests/integration/test_kinesis.py

```diff
@@ -71,6 +72,7 @@ def test_subscribe_to_shard(self):
         result = client.create_stream(StreamName=stream_name, ShardCount=1)
         sleep(1)
         result = client.register_stream_consumer(StreamARN=stream_arn, ConsumerName='c1')['Consumer']
+        sleep(1)
```

Comment:
kinesis-mock applies KINESIS_LATENCY to RegisterStreamConsumer actions so I added a sleep here.